lib/extend.js

```diff
@@ -54,8 +54,18 @@ function extend(opts) {
     if (!_.isArray(options.source))
       throw new Error('source must be an array');
 
-    var props = combineJSON(options.source, {deep: true});
-    to_ret.properties = deepExtend({}, to_ret.properties, props);
+    var props = combineJSON(options.source, true, function Collision(opts) {
+      if (options.log) {
+        var str = `Collision detected at: ${opts.path.join('.')}! Original value: ${opts.target[opts.key]}, New value: ${opts.copy[opts.key]}`;
+        if (options.log === 'warn') {
```

Comment:
I couldnt find the log option inside the doc. I was gonna write JS script but got saved.

Reply:
@NicestRudeGuy Hi, How to you config this? Could you share a example? Thanks

Reply:
I think the type declaration about this option is wrong

Reply:
@EiffelFly  if you mean to stop the StyleDictinary from executing when there is collision you can just add 
`"log": "error"` to its config file